test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -93,3 +97,390 @@ def test_exclusive_job_not_suspended(self):
+        except:
+            pass
+        a = {'queue_type': 'e',
+             'started': 'True',
+             'enabled': 'True',
+             'Priority': 170}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, "expressq3")
+
+        # Count the compute nodes
+        vlist = []
+        try:
+            vnl = self.server.filter(
+                VNODE, {'resources_available.vntype': 'cray_compute'})
+            vlist = vnl["resources_available.vntype=cray_compute"]
+        except Exception:
+            sys.exc_clear()
+            self.logger.info("There are no cray_compute vnodes present.")
+        numvnl = len(vlist)
+
+        nodes = self.server.status(NODE)
+        for n in nodes:
+            if 'resources_available.vntype' in n:
+                if n['resources_available.vntype'] == 'cray_compute':
+                    cpus = n['resources_available.ncpus']
+                    ncpus = int(cpus)
+
+        j1 = Job(TEST_USER, {ATTR_l + '.select': '%d:ncpus=%d' % (numvnl, ncpus),
+                             ATTR_l + '.walltime': 3600})
+        j1id = self.server.submit(j1)
+        self.server.expect(JOB, {ATTR_state: 'R', ATTR_substate: 42}, id=j1id)
+        qstat = self.server.status(JOB, 'exec_vnode', id=j1id)
+        vname = qstat[0]['exec_vnode'].partition(':')[0].strip('(')
+
+        j2 = Job(TEST_USER, {ATTR_l + '.select': '%d:ncpus=%d' % (numvnl, ncpus),
+                             ATTR_l + '.walltime': 3600,
+                             ATTR_q: 'expressq'})
+        j2id = self.server.submit(j2)
+        self.server.expect(JOB, {ATTR_state: 'S'}, id=j1id)
+        self.server.expect(JOB, {ATTR_state: 'R', ATTR_substate: 42}, id=j2id)
+
+        j3 = Job(TEST_USER, {ATTR_l + '.select': '%d:ncpus=%d' % (numvnl, ncpus),
+                             ATTR_l + '.walltime': 3600,
+                             ATTR_q: 'expressq2'})
+        j3id = self.server.submit(j3)
+        self.server.expect(JOB, {ATTR_state: 'S'}, id=j2id)
+        self.server.expect(JOB, {ATTR_state: 'R', ATTR_substate: 42}, id=j3id)
+
+        j4 = Job(TEST_USER, {ATTR_l + '.select': '%d:ncpus=%d' % (numvnl, ncpus),
+                             ATTR_l + '.walltime': 3600,
+                             ATTR_q: 'expressq3'})
+        j4id = self.server.submit(j4)
+        self.server.expect(JOB, {ATTR_state: 'S'}, id=j3id)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=j4id)
+
+        self.server.delete(j4id)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=j3id)
+
+    def test_job_accounted_limit_usage(self):
+        """
+        Test that a job is not incorrectly accounted limit usage
+        """
+        try:
+            self.server.status(QUEUE, id='expressq', logerr=False)
+            self.server.manager(MGR_CMD_DELETE, QUEUE, None, 'expressq')
+        except (PbsStatusError, PbsManagerError):
+            pass
+
+        a = {'queue_type': 'execution',
+             'priority': 150,
+             'enabled': 1,
```

Comment:
There is no harm in having is as is, but my preference is that you change it to 'true' to match previous test cases